javascript/new-src/Gruntfile.js

```diff
@@ -14,28 +49,20 @@ module.exports = function(grunt) {
         }
       }
     },
+    concat: {
+      options: {
+        stripBanners: true
+      },
+      build: {
+        src: targets.js,
+        dest: '../../olad/www/new/js/app.js'
+      }
+    },
     uglify: {
       build: {
         files: [{
           dest: '../../olad/www/new/js/app.min.js',
-          src: [
-            'src/app.js',
-            'src/controllers/menu.js',
-            'src/controllers/patch_universe.js',
-            'src/controllers/rdm_universe.js',
-            'src/controllers/universe.js',
-            'src/controllers/fader_universe.js',
-            'src/controllers/keypad_universe.js',
-            'src/controllers/plugins.js',
-            'src/controllers/add_universe.js',
-            'src/controllers/plugin_info.js',
-            'src/controllers/setting_universe.js',
-            'src/controllers/header.js',
-            'src/controllers/overview.js',
-            'src/constants.js',
-            'src/factories/ola.js',
-            'src/filters/start_form.js'
-          ]
+          src: ['../../olad/www/new/js/app.js']
```

Comment:
I intend to create (and keep) the app.js file for debugging purpouses, I will add makefile/server handlers for it